cgi/org.pl

```diff
@@ -358,29 +358,28 @@
 	if ($type eq "edit") {
 
 		store_org($org_ref);
-		$template_data_ref->{result} = lang("edit_org_result");
-
-		$template_data_ref->{profile_url} = canonicalize_tag_link("editors", "org-" . $orgid);
-		$template_data_ref->{profile_name} = sprintf(lang('user_s_page'), $org_ref->{name});
 	}
 	elsif ($type eq 'user_delete') {
 
 		if (is_user_in_org_group($org_ref, $User_id, "members")) {
-			my $groups_ref = ['admins','members'];
+			my $groups_ref = ['admins', 'members'];
 			my $user_to_remove = single_param('user_id');
+			# remove the user from org
 			remove_user_from_org($orgid, $user_to_remove, $groups_ref);
 
 			# Reset the 'org' field of the user
 			my $user_ref = retrieve_user($user_to_remove);
 			delete $user_ref->{org};
 			delete $user_ref->{org_id};
-			my $user_file = "$data_root/users/" . get_string_id_for_lang("no_language",$user_to_remove) . ".sto";
+			my $user_file = "$data_root/users/" . get_string_id_for_lang("no_language", $user_to_remove) . ".sto";
```

Comment:
Could we make this all a sub in Users.pm ?

This would ease testing and we may want to reuse that.

Ideally, all business rules code should be in modules (and cgi should only deal with http/html/form mechanics).